resources/views/community/components/developer/sets-page.blade.php

```diff
@@ -0,0 +1,153 @@
+@props([
+    'user' => null,
+    'consoles' => [],
+    'games' => [],
+    'sortOrder' => 'title',
+    'filterOptions' => [],
+    'userProgress' => null,
+])
+
+<x-app-layout
+    pageTitle="{{ $user->User }} - Developed Sets"
+    pageDescription="Games developed by {{ $user->User }}"
```

Comment:
Apologies for leaving comments so sporadically, I'm basing the recent unlocks work on a lot of your work here.

```suggestion
    pageDescription="View achievement sets developed by {{ $user->User }} for various games on RetroAchievements"
```

This meta description is slightly better for SEO (this page will almost certainly get crawled).